src/relay/backend/compile_engine.cc

```diff
@@ -689,7 +689,7 @@ class CompileEngineImpl : public CompileEngineNode {
       cache_node->funcs = (*f)(cfunc->schedule, all_args, cache_node->func_name, key->source_func);
     } else {
       using tvm::transform::PassContext;
-      With<PassContext> fresh_pass_ctx_scope(PassContext::Create());
+      With<PassContext> fresh_pass_ctx_scope(static_cast<PassContext>(PassContext()));
```

Comment:
I don't think static cast would be necessary here, With<PassContext> fresh_pass_ctx_scope(PassContext()); should suffice once we defined the copy constructor

Reply:
@tqchen : Thanks! I tried with copy constructor [using](https://github.com/apache/incubator-tvm/blob/9151a519b9495483a3d2bf552c8d6cf02e2302a3/include/tvm/runtime/object.h#L686), but it did not work out.

Reply:
Would be interesting to see what error message you got. As copy constructor is clearly defined in there.
We can also use `With<PassContext> fresh_pass_ctx_scope();`, which invokes the default constructor(in this case the new pass context.

Reply:
@tqchen : Sorry for late reply! If i dont use static_cast [the testcase fails with additional assert](https://github.com/apache/incubator-tvm/blob/6ae439c8c58dd0118d2f2c5d1c4bcb650df47104/tests/python/relay/test_pass_manager.py#L555) count.

Reply:
@tqchen : I have changed it to default constructor now, please check. Thanks!

Reply:
> @tqchen : I have changed it to default constructor now, please check. Thanks!

@tqchen : Sorry! This change does not work. I made a mistake in code build, cause of that it got passed last time! So i have to use static_cast for the test case to pass.

Reply:
@tqchen : Any thoughts on this comment?